README.md

```diff
@@ -31,6 +31,15 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Abort on errors
+
+In the default setup, all tasks are continued even if any of the task produces an error. Furthermode, calling `grunt […]` will always exit with code 0.
```

Comment:
```suggestion
In the default setup, all tasks are continued even if one of the tasks produces an error. Furthermore, calling `gulp […]` will always exit with code 0.
```

Reply:
Thanks! Fixed it, please see the latest version of readme.md